Magento_Sales/templates/order/order_date.phtml

```diff
@@ -1,6 +1,6 @@
 <div class="dashboard__order-date margin-bottom-sm">
     <span>
-        <?= __('Created') ?>:
+        <?= $block->escapeHtml(__('Created')) ?>:
     </span>
     <date>
         <?= $block->escapeHtml(__($block->formatDate($block->getOrder()->getCreatedAt(), \IntlDateFormatter::LONG))) ?>
```

Comment:
here we can remove escapeHtml and add noEscape. In original file escapeHtml is used but only because if translations. Here also we can remove translations `__()` method